src/NuGet.Clients/NuGet.VisualStudio.Common/VsProjectTypes.cs

```diff
@@ -37,5 +42,7 @@ public static class VsProjectTypes
 
         // All unloaded projects have this Kind value
         public const string UnloadedProjectTypeGuid = "{67294A52-A4F0-11D2-AA88-00C04F688DDE}";
+
+        public const string NuGetSolutionSettingsFolder = ".nuget";
     }
```

Comment:
Is there really no better place for this?
I really dislike this (:

Reply:
this is not used, so I will get rid of it.